### DISCUSSION TITLE
Editing OnlyOffice documents not working initially

### DISCUSSION THREAD

**Discussion by IwisGiwis** [SafeExamBrowser/seb-win-refactoring#1055] [Q&A]
Dear all,

In some exam scenarios at our university students have to use OnlyOffice for editing documents. To do that students click on edit in the LMS (OpenOlat) and a new browser window opens with OnlyOffice (version 8.1.1.26). In other browsers like Firefox (e. g. version 133.0) users can then edit the document but that does not work with SEB. When using SEB the document can't be edited unless the user either:

Clicks the reload button in the SEB window where OnlyOffice is opened.

Sets the mouse focus to the first opened SEB window with Openolat, which is in the background, and then sets the mouse focus back to the front SEB window with the OnlyOffice document.

We know of another institute that has the same problem.
Are you aware of that problem? Should I provide log files?

Best regards,
Stefan

**Comment by HeikoUHH** [comment]
At my faculty, we are just starting to introduce digital exams with the same setting (SEB with Openolat and OnlyOffice) and this problem also occurs here. Therefore, we would also be very happy if this phenomenon could be resolved.

Thank you very much!

**Comment by dbuechel** [comment]
Please verify whether this behavior reproduces with the CEF sample application (see https://cef-builds.spotifycdn.com/index.html). If so, it is an upstream issue in the browser engine and must be resolved there. If not, we'd need a demo access or alike so that we can reproduce the issue ourselves.

And yes, in either case please provide the log files of Safe Exam Browser, of which especially the client log file is relevant.

**Reply by IwisGiwis** [reply]
Thanks for the info. The behaviour is the same with the CEF version 121.3.15+g4d3b0b4+chromium-121.0.6167.184 and that seems to be the same version that SEB 3.8 and I think SEB 3.7 uses. The current stable version 131.3.1+gcb062df+chromium-131.0.6778.109 does not have this problem.

[2024-12-12_11h14m11s_Runtime.log](https://github.com/user-attachments/files/18109644/2024-12-12_11h14m11s_Runtime.log)
[2024-12-12_11h14m11s_Browser.log](https://github.com/user-attachments/files/18109645/2024-12-12_11h14m11s_Browser.log)
[2024-12-12_11h14m11s_Client.log](https://github.com/user-attachments/files/18109646/2024-12-12_11h14m11s_Client.log)

**Reply by dbuechel** [reply]
Okay, then you're in luck and the issue appears to have been resolved upstream. We'll update the browser engine to the latest available version for the next release version 3.9.0.

**Reply by IwisGiwis** [reply]
Thanks for updating and also the great support!

**Reply by IwisGiwis** [reply]
Hi,

Sadly the behavior persists with SEB version 3.9.0. The problem does not occur with Chromium version 131.0.6778.205. So it does not seem to be an upstream issue. 

If you want to investigate further into this issue do you only need access to our LMS where you can open OnlyOffice documents or do you need additional access to the OnlyOffice server for logs?

Best regards,
Stefan

**Reply by dbuechel** [reply]
If you think that it isn't an upstream issue, then yes, we'd need a demo access or alike so that we can reproduce the issue ourselves. Please continue the discussion under #1133 and provide us with a demo exam and step-by-step instructions on how we can reproduce the behavior.
